src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/JdbcTestIT.java

```diff
@@ -24,6 +24,22 @@
     @Override
     protected void init() throws Exception {
         loadIndex(Index.ONLINE);
+        loadIndex(Index.PEOPLE);
+    }
+
+    public void testPercentilesQuery() {
+        JSONObject response = executeJdbcRequest(
+                "SELECT percentiles(age, 25.0, 50.0, 75.0, 99.9) age_percentiles" +
+                        " FROM elasticsearch-sql_test_index_people");
```

Comment:
np: indentation w.r.t SELECT. Similarly for `testGroupByInQuery`.

Reply:
Done